.github/workflows/checks.yaml

```diff
@@ -29,6 +29,9 @@ jobs:
       with:
         go-version: '${{ env.golang-version }}'
         check-latest: true
+    - name: Install gsed on macOS
```

Comment:
I'm not sure to understand why this is needed since the GitHub action runs fine on MacOS...